interface/src/ui/PreferencesDialog.cpp

```diff
@@ -400,6 +400,16 @@ void setupPreferences() {
         preference->setItems(items);
         preferences->addPreference(preference);
     }
+    {
+        auto getter = [myAvatar]()->float { return myAvatar->getHMDYawSpeed(); };
```

Comment:
I think this should be:

```suggestion
        auto getter = [&myAvatar]()->float { return myAvatar->getHMDYawSpeed(); };
```

Without the reference this will copy the myAvatar object